io.ossia.score.yml

```diff
@@ -0,0 +1,281 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+rename-desktop-file: ossia-score.desktop
+rename-icon: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Required for USB, Webcam devices support as well as raw GPIO I/O
+  - --device=all
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Used by X11
+  - --share=ipc
+
+  # Network access
+  - --share=network
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Audio support through pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+  # Ignore ALSA configuration
+  - --env=ALSA_CONFIG_PATH=
```

Comment:
Why do you need to ignore config from runtime?

Reply:
I saw this in all the pro-audio flatpak I saw - Ardour, Mixxx, Bitwig. Ardour has a comment that it is necessary for sound to work on SuSE : https://github.com/flathub/org.ardour.Ardour/blob/master/org.ardour.Ardour.json#L19C23-L21C31

Reply:
It was done a bit blindly. I don't have SUSE to reproduce. And then it get copied.

Reply:
rationale seems to be given here: https://github.com/flatpak/flatpak/issues/1875#issuecomment-520401097

Reply:
removed anyways, hopefully it won't break for anyone !